android/src/main/java/com/tanguyantoine/react/MusicControlModule.java

```diff
@@ -361,7 +379,8 @@ private Bitmap loadArtwork(String url, boolean local) {
         Bitmap bitmap = null;
 
         try {
-            if(local) {
+            // If we are running the app in debug mode, the "local" image will be served from htt://localhost:8080, so we need to check for this case and load those images from URL
+            if(local && !url.startsWith("http")) {
```

Comment:
This works, but it wouldn't work for other schemes. Take a look what I've done in my own module: https://github.com/Guichaguri/react-native-track-player/blob/dev/android/src/main/java/guichaguri/trackplayer/logic/track/Track.java#L171-L181